src/var/mash.jl

```diff
@@ -0,0 +1,34 @@
+# MinHash
+# ========
+#
+# Functions to MASH distance for MinHash sketches
+#
+# see DOI: 10.1186/s13059-016-0997-x
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+function jaccarddistance(sketch1::MinHashSketch, sketch2::MinHashSketch)
+    d = length(setdiff(sketch1.sketch, sketch2.sketch))
```

Comment:
That would be great if it's possible to eke out a bit more speed. I drew a draft version of a function that does the same job as `setdiff` but is specific to sorted arrays, when I checked out `setdiff` with `@less` the other day, I can make a gist out of it if it helps.